build/azure-pipelines.yml

```diff
@@ -66,6 +66,23 @@ jobs:
       continueOnError: true
       condition: always()
 
+- job: RichNav
+  pool:
+    vmImage: 'vs2017-win2016'
+  variables:
+    _configuration: Debug
+  timeoutInMinutes: 20
+  steps:
+    - script: $(Build.SourcesDirectory)\build\CIBuild.cmd -configuration $(_configuration) -prepareMachine
```

Comment:
Reading through the docs I don't think we need to build

Reply:
reading through _the code_ I know we don't need to build. but we do need to restore the project.

Today `CIBuild.cmd` does the following by default
```console
-restore -build -test -pack -ci
```

We only need to do
```console
-restore -ci
```

So we should probably have a new script named `CIRestore.cmd` whose contents look like this
```console
echo off
powershell -ExecutionPolicy ByPass %~dp0Build.ps1 -restore -ci %*
exit /b %ErrorLevel%
```

which we call in the yaml file like so:
```yaml
  steps:
    - script: $(Build.SourcesDirectory)\build\CIRestore.cmd -configuration $(_configuration) -prepareMachine
```